.github/workflows/ci.yml

```diff
@@ -5,6 +5,18 @@ on:
     branches-ignore:
       - main
 
+  # When Changesets opens a PR it does not trigger GitHub actions,
```

Comment:
Hmm... Maybe we need to add this to the Storybook tests flow too? 🤔 It will get skipped, but the workflow still needs to be triggered.

Alternatively we could create a label? Although I guess this saves you from multiple steps.

Reply:
Oh, cause now it's separated, yeah.